### ISSUE TITLE
counter.inc(0) will increment by 1

### ISSUE THREAD

**Issue by tvcutsem** [felixge/node-measured#18]
In my application I'm dynamically incrementing a counter based on the size of a collection, e.g.

```js
var counter = new measured.Counter();
...
counter.inc(array.length);
```

I got wrong counts, and the issue is that if array.length is 0, the counter is still incremented by 1.

This is due to the inc and dec functions using the `||` logical-or operator to fill in a default value if the parameter `n` is not passed. However, `0 || 1` will evaluate to `1`, which is probably not what was intended. It would be safer to update the inc and dec functions to explicitly test whether `n` is `undefined` (i.e. `n === undefined ? 1 : n`)

**Comment by mantoni** [comment]
Ah, good point. Fancy making it a pull request?

**Comment by mantoni** [comment]
Released in `1.0.1`.

**Comment by tvcutsem** [comment]
Thanks for the quick fix!
